java/java.guards/src/org/netbeans/modules/java/guards/JavaGuardedReader.java

```diff
@@ -194,8 +194,9 @@ public JavaGuardedReader(JavaGuardedSectionsProvider provider) {
                         charBuffPtr += MAGICLEN;
                     } else {
                         i += toNl;
-                        Arrays.fill(charBuff,charBuffPtr,charBuffPtr+sectionSize,' ');
```

Comment:
I guess that the whitespaces were added because we cannot assign an empty string (`''`) to char. Of course, I could be wrong.